### DISCUSSION TITLE
Sd is not taking the data from the code

### DISCUSSION THREAD

**Discussion by Manthu-Patil** [ravdsn/RFID-Based-Attendance-System-using-Arduino#3] [General]
The data is not been read into sd card the else part is getting executed can you plzz help me out

**Comment by ravinduu** [comment]
Hi Manthu-Patil, Thanks for using our project. But this is a very old project. We are not currently maintaining this. So we are unable to help you with this issue.

**Reply by kanissh** [reply]
yes @Manthu-Patil, we recommend that you search the arduino hub - https://create.arduino.cc/projecthub
you will for sure find answers
